joint_limits:
  left_wheel_joint:
    has_position_limits: false
  right_wheel_joint:
    has_position_limits: false